tasks/__init__.py

```diff
@@ -35,6 +36,16 @@ def lint_all(context: Context) -> None:
     backend.lint(context)
 
 
+@task
+def pull(context: Context) -> None:
+    """Pull the latest changes from Github and update the submodule to the proper commit."""
+    commands = ["git pull", "git submodule update"]
```

Comment:
```suggestion
    commands = ["git pull --recurse-submodules"]
```

Seems to do the same thing?

Reply:
I'm not sure ... 
I was under the impression that `git pull --recurse-submodules` will pull the latest commit in the submodules if we are tracking a branch while `git submodule update` will checkout the right commit.


we could do more testing but either way it feels like a `inv pull` is shorter to execute :)